site/docs/components/link.mdx

```diff
@@ -0,0 +1,129 @@
+---
+name: Link
+menu: Components
+route: /components/link>
+---
+
+import { Playground } from "docz";
+import { Link } from "hds-react";
+import LargeParagraph from "../../src/components/LargeParagraph";
+
+# Link
+
+<LargeParagraph>
+    Links are used as navigational elements and can be used on their own or inline with text. They provide a lightweight option for navigation.
+</LargeParagraph>
+
+## Principles
+- Too many links will clutter a page and make it difficult for users to identify their next steps. This is especially true for inline links, which should be used sparingly.
+- Use links when you want users to:
+    - Navigate to a different page
+    - Navigate to an entirely different site
+    - Jump to an element on the same page
+    - Link to emails or phone numbers
+- HDS Link supports a link with an external icon. Use the `external` prop when you wish to navigate to an entirely different site.
+- HDS Link supports a link that opens in a new tab. Use the `openInNewTab` prop when you wish to open the link in a new tab.
+
+
+## Accessibility
+The HDS link component is primarily a native HTML hyperlink navigational element. The Enter key activates the link and causes the user agent to move focus to the link destination.
+
+- Remember to tell users if the link is going to open an entirely new site. In HDS React Link this feature is provided through external links (prop `external`). There is a default aria-label for screen readers. This can be overwritten by using the `openInExternalDomainAriaLabel` prop.
+In HDS Core, set a proper `aria-label` to the anchor tag as shown in the examples below.
+
+- Remember to tell users if the link is going to open in a new tab. In HDS React Link this feature is provided through prop `openInNewTab`. There is a default aria-label for screen readers. This can be overwritten by using the `openInNewTabAriaLabel` prop.
+In HDS Core, set a proper `aria-label` to the anchor tag as shown in the examples below.
+
+- Some assistive technologies provide a list of all links on the page to its user. If the text of the links do not clearly give the purpose, or if multiple links with the same name point to different targets (e.g., “read more”), users are forced to locate the link on the page and search surrounding information for context.
```

Comment:
"the text of the links **does** not clearly"
or
"the **texts** of the links do not clearly"

Reply:
- [ ] do not -> does not